### PR TITLE
FOGL-2264 Update build instructions to create static libraries for fr…

### PR DESCRIPTION
**Pull request by MarkRiddoch**
…eeopcua

Note, if you have previously done a build of freeopcua you will need to delete the build directory before following these instructions.

### CODE REVIEW

**Comment by ashish-jabble** [comment]
Fresh build with applying above changes in `freeopcua` as suggested, but the problem is still there.

[mypatch.patch.zip](https://github.com/foglamp/foglamp-south-opcua/files/2812332/mypatch.patch.zip)

**Comment by MarkRiddoch** [comment]
> Fresh build with applying above changes in `freeopcua` as suggested, but the problem is still there.
> 
That's odd. Could you run the command

`ldd libopcua.so`

and let me have the results please.

File: README.rst [hunk 1]
```diff
@@ -29,6 +29,23 @@ and set it to OFF.
 
   option(SSL_SUPPORT_MBEDTLS "Support rsa-oaep password encryption using mbedtls library " OFF)
 
+.. code-block:: console
+
+The build options for the OPCUA libraries must be changed to create static libraries. To do this
+find the occurances of the add_library directive for opcuaclient, opcuacode and opcuaprotocol
```
Comment (ashish-jabble) [inline README.rst]: `occurrences`

**Review by ashish-jabble — APPROVED** [review]
Now I am able to discover plugin via GUI. LGTM!

**Pull request merged**